wgpu/gui/_offscreen.py

```diff
@@ -8,6 +8,12 @@ class WgpuOffscreenCanvas(WgpuCanvasBase):
     The resulting texture view is passed to the ``present()`` method.
     """
 
+    def __init__(self, *args, **kwargs):
+        # we don't pass on constructor arguments; code may
+        # be assuming we are relying on e.g. qt or wx or jupyter
+        # but in the offscreen case we are definitely not
```

Comment:
I needed to do this because some examples pass e.g. `title="cube example"` which obviously isn't supported by `object.__init__()` which is the base class of offscreen canvases.